radix-engine-tests/tests/blueprints/account_locker.rs

```diff
@@ -196,7 +196,7 @@ fn store_can_only_be_called_by_storer_role() {
                         ACCOUNT_LOCKER_STORE_IDENT,
                         AccountLockerStoreManifestInput {
                             bucket,
-                            claimant: account.into(),
+                            claimant: account.into_manifest_address().into(),
```

Comment:
It might be nice to add a `From<ComponentAddress> for Global<ManifestComponentAddress, _>`